src/states_screens/help_screen_5.cpp

```diff
@@ -0,0 +1,84 @@
+//  SuperTuxKart - a fun racing game with go-kart
+//  Copyright (C) 2009-2015 Marianne Gagnon
```

Comment:
Better grammar would be "the SuperTuxKart Team".  Not a huge deal, but a bit nicer to look at.